tests/test_optimize.py

```diff
@@ -78,7 +77,10 @@ def test_optimize(
         storage = get_writable_storage(storage_key)
         cluster = storage.get_cluster()
         clickhouse = cluster.get_query_connection(ClickhouseClientSettings.OPTIMIZE)
-        table = storage.get_table_writer().get_schema().get_table_name()
+        table = (
+            storage.get_table_writer().get_schema().get_table_name().split("_")[0]
+            + "_local"
+        )
```

Comment:
The table name suffixes `_dist` and `_local` are a convention but there's no guarantee those are the actual names. There is a `get_local_table_name()` function you can use instead.

Reply:
That's a good catch; modified it to use `get_local_table_name()`.